src/components/IconButton/IconButton.jsx

```diff
@@ -76,32 +77,34 @@ const IconButton = forwardRef(
     }, [disabled, disabledReason, tooltipContent, ariaLabel]);
 
     return (
-      <ToolTip content={content} referenceWrapperClassName={styles.referenceContainer}>
-        <Button
-          onClick={onClick}
-          disabled={disabled}
-          color={color}
-          kind={kind}
-          ariaLabel={buttonAriaLabel}
-          ref={mergedRef}
-          id={id}
-          dataTestId={dataTestId || getTestId(ELEMENT_TYPES.ICON_BUTTON, id)}
-          className={cx(className)}
-          noSidePadding
-          active={active}
-          style={overrideStyle}
-          insetFocus={insetFocus}
-        >
-          <Icon
-            icon={icon}
-            iconType={Icon.type.SVG}
-            iconSize={iconSize}
-            ignoreFocusStyle
-            className="icon-button-padding"
-            clickable={false}
-          />
-        </Button>
-      </ToolTip>
+      <div className={cx(wrapperClassName, styles.wrapper)}>
```

Comment:
maybe iconButtonWrapper will be more clear?

Reply:
Also let's do it conditional for not break exist components which use this component already
something like: const IconButtonWrapper = wrapperClassName ? "div" : Fragment